src/icons/cc-amex.svg

```diff
@@ -1,7 +1,4 @@
-<svg>
-  <title>iconTitle</title>
-  <desc>iconDesc</desc>
-  <style></style>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
In prior files I assumed that you were just adding this in with new files, but why are you adding this here?

These, and other default values, are set from this config
https://github.com/AlaskaAirlines/OrionIcons/blob/master/src/data/icons.json#L2-L13

I see your changes updating how title and description work, but nothing in regards to this?

Reply:
Running this locally I see that the hard-coded stuff in the SVG file is replaced with the values from the data, but this still doesn't make sense as to why you added this to this and other SVGs?

Reply:
It wasn't intentional to add these attributes. I was handed a zip folder with all the svgs containing these attributes. While I agree these values are extra and "dirty" the data I believe it may be more work to scrub it manually on every instance of introducing a new `svg` instead of building a system that is tolerant enough to handle it.

Reply:
Here is my issue. We have no guarantee that whoever is building out the SVGs will deliver clean versions. The code you have may catch some issues, but cannot perceive all. 

This is why I made it a specific part of the contribution process that the SVGs be sanitized prior to the PR being accepted. 

I am sorry that you had to do a large volume of icons, but I don't see that being the norm, but simply a case of quick ramp-up.

Reply:
Post chore story: clean up after @Kiricon